sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Scenario.Tests/tests/MultipleStorageAccountsEndToEndTests.cs

```diff
@@ -222,8 +222,9 @@ public async Task InitializeAsync(WebJobsTestEnvironment testEnvironment)
 
                 BlobServiceClient1 = new BlobServiceClient(testEnvironment.PrimaryStorageAccountConnectionString);
                 BlobServiceClient2 = new BlobServiceClient(testEnvironment.SecondaryStorageAccountConnectionString);
-                QueueServiceClient1 = new QueueServiceClient(testEnvironment.PrimaryStorageAccountConnectionString);
-                QueueServiceClient2 = new QueueServiceClient(testEnvironment.SecondaryStorageAccountConnectionString);
+                var queueOptions = new QueueClientOptions() { MessageEncoding = QueueMessageEncoding.Base64 };
```

Comment:
Why do you have to do this considering base64 is the default?

Reply:
I was fixing other places like that in tests, where SDK clients were used in tests to set up some resources directly. So must have pattern-matched here as well. But yes, it doesn't seem to do anything useful here as SDK seems to be only used to create queue in this test class. I'll give it a try and remove if that's true.

Reply:
While looking closer and trying out. The SDK clients are used directly to send and receive messages in this test class.
So while extension is base64 by default the SDK isn't so this is to keep direct usage of SDK clients in parity with extension.